pkg/resources/prometheusRules.go

```diff
@@ -63,6 +63,17 @@ func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8scli
 		}
 	}
 
+	rule := &monitoringv1.PrometheusRule{}
+	err := client.Get(ctx, k8sclient.ObjectKey{
+		Name:      "ksm-monitoring-alerts",
+		Namespace: "redhat-rhoam-middleware-monitoring-operator",
+	}, rule)
+	if !errors.IsNotFound(err) {
+		if err := client.Delete(ctx, rule); err != nil {
+			r.Log.Info("Alert \"ksm-monitoring-alerts\" was deleted")
+		}
+	}
+
```

Comment:
I would like to add the removal of the rate limiting soft alerts in to this PR, the changes required are below. This is will address the jira: https://issues.redhat.com/browse/MGDAPI-1705

```suggestion
	err := client.Get(ctx, k8sclient.ObjectKey{
		Name:      "rate-limit-soft-limits,
		Namespace: "redhat-rhoam-marin3r",
	}, rule)
	if !errors.IsNotFound(err) {
		if err := client.Delete(ctx, rule); err != nil {
			r.Log.Info("Alert \"rate-limit-soft-limits\" alert was deleted")
		}
	}

```

Reply:
included removal of the soft limits alert